client/state/ui/editor/video-editor/actions.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+import {
+	VIDEO_EDITOR_POSTER_UPDATE,
+	VIDEO_EDITOR_POSTER_UPDATE_FAILURE,
+	VIDEO_EDITOR_POSTER_UPDATE_SUCCESS,
+	VIDEO_EDITOR_SCRIPT_LOAD_ERROR,
+	VIDEO_EDITOR_STATE_RESET,
+	VIDEO_EDITOR_STATE_RESET_POSTER,
+	VIDEO_EDITOR_VIDEO_HAS_LOADED,
```

Comment:
@dbtlr I like to do this for objects so that if you need to add another property later, you don't get the extra diff on the preceding line. I see this being done occasionally in Calypso, but it's not part of the style guide.